conda/auxlib/entity.py

```diff
@@ -252,7 +252,6 @@
 from .collection import AttrDict
 from .exceptions import Raise, ValidationError
 from .ish import find_or_raise
-from .logz import DumpEncoder
```

Comment:
If we are concerned with downstream `conda.auxlib.entity.DumpEncoder` imports we can also do:

```python
deprecated.constant(
    "26.3",
    "26.9",
    "DumpEncoder",
    json.CondaJSONEncoder,
    addendum="Use `conda.common.serialize.json.CondaJSONEncoder` instead.",
)
```

Reply:
That seems maybe overley conservative to me. Do we really want to be deprecating every time we want to remove an import from a module?

If you think it's important, can do.

Reply:
Let's skip adding that in here.